.github/workflows/ci.yaml

```diff
@@ -6,8 +6,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - name: docker://python:3.8-bullseye
-      uses: docker://python:3.8-bullseye
+    - name: docker://python:3.10.13
+      uses: docker://python:3.10.13
```

Comment:
Not sure why the previous team used bullseye, but we should probably keep it the same if we don't know why they used it. There's a [3.10-bullseye](https://github.com/docker-library/python/blob/936a2ed61e9930bfec0de767e78877e29d288322/3.10/bullseye/Dockerfile)

https://hub.docker.com/_/python/

Reply:
I looked into this. Future versions in development, but not yet stable, are “Bullseye” and “Bookworm.” I don't think we should be using these images regardless of what the previous team did. That base Python image that Ed chose is based on the most recent stable image of Debian, which seems far safer to me than an unstable version.

Reply:
I can't tell why they ever used bullseye. Rescinding this requested change. The API uses alpine, so I could see narrowing it down to that but identifying which is beyond the scope of this work and can be determined at a later time. python3.10.13 works and we'll call that good.